src/users/users.controller.ts

```diff
@@ -13,25 +13,30 @@ import { GuardMiddleware } from '../common/guard.middleware';
 
 @injectable()
 export class UsersController extends BaseController implements IUsersController {
+	private context = 'users';
```

Comment:
let's remove it. Use please class name

Reply:
I will keep this parameter for transparent logging router works